nncf/experimental/torch/fx/transformations.py

```diff
@@ -802,12 +803,26 @@ def apply_quantization_transformations(model: torch.fx.GraphModule) -> None:
     # to make it easier for algorithms to work
     # with the target graph BatchNorm operations
     # are being fused
+    fold_constant_except_qdq(model)
```

Comment:
Can constant folding also be applied for weights compression?

Reply:
Can you provide an use case / model to justify this change?